kobotoolbox/kpi
# feat(organizations)!: Restrict project usage access to admins and organization owners

### 📣 Summary
Restricted access to project usage data to admins and organization owners only.


### 📖 Description
This PR introduces stricter access control for project usage data, limiting visibility and management to admins and organization owners. This change ensures that only authorized users can access and manage usage metrics, protecting organizational data from unauthorized access.


### 💭 Notes
The permission classes for organizations have been refactored to provide more granular control over nested objects related to organizations. 

### 👀 Preview steps

1. Create 3 accounts
2. Go to admin > Organizations. 
3. Set user1's organization as a MMO organization
4. Add user2 to organization as an admin
5. Add user3 to organization as a regular member
6. Go to `/api/v2/organizations/<organization_id>/asset_usage/`
6. In `main`, user1, user2 and user3 should access the page
7. With this PR, only user1 and user2 should access it. `user3` should receive a 403
